### PR TITLE
Milestone - Welcome to Microverse

### PR DESCRIPTION
**Pull request by morehwachege**
- Add index page
- CSS stylesheet linked to index
- Add GitHub workflows for HTML/CSS linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,111 @@
...
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Please omit the comment. This makes your code base clean. The section is  a guide to help you draft the README

File: README.md [hunk 2]
```diff
@@ -1 +1,111 @@
...
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Hello Microverse ](#-hello-microverse-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [📝 License ](#-license-)
+
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] All sections of the README has to be added. I've attached a screen shot of how it should be 

# View:
image

File: README.md [hunk 3]
```diff
@@ -1 +1,111 @@
...
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] This line is a guide to you, and should not be added to the code base. kindly omit it

File: test.md [hunk 4]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (Shedrack-Sunday) [inline test.md]: - [ ] This file is meant to be ignored by git. kindly fix that 👍

File: README.md [hunk 5]
```diff
@@ -1 +1,111 @@
...
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Please delete the logo image in this section. You can either add your custom logo or leave the space blank. It was placed there to give you a guide on how to fix yours. This is a good viewpoint practice for future projects 😉

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @morehwachege  👋 


# CHANGES REQUIRED ♻️
Great work on the project 👏
You've done well implementing some of the features needed, but there are still some which aren't been addressed yet.



Highlights
✅All linters are checked

Required Changes ♻️

- [ ] Every section of the [README TEMPLATE](https://github.com/microverseinc/readme-template) has to be added. you can customize them under the header depending on the project, but the sections have to be available 
- [ ] Please always try to give a descriptive name for your branches . `feature` is too generic, it would be helpful when you have a large code base or working in a team. it s needed to talk about changes. please take note. 

- [ ] Kindly remove descriptions on the README under the `>` sections. They're guides and should be personalized as much as possible 

_check the comment section for a detailed review_

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: test.md (hunk 4 above)
Comment (didierganthier) [reply test.md]: Hey 👋🏽  this is the last thing you need to handle 1️⃣  this file is supposed to be ignored by git, to fix that delete the file you have on Git, pull from this branch `feature`, and create this file again locally, this time it won't be added to git and your project will be approved. 🚀

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @morehwachege ,

🎉 Great job on addressing the feedback from the previous review! You've made some solid progress in implementing the requested changes. However, there are still a few areas that need some attention before we can move on to the next project.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @morehwachege , 

I am Favour 👩🏽 , your reviewer for this PR.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**